mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/DefaultEntityFactory.java

```diff
@@ -102,21 +102,36 @@ public EntityDefinition getDefinition(TypeElement classElement) {
           continue;
         }
         ExecutableElement getMethod = (ExecutableElement) child;
-        String getMethodName = getMethod.getSimpleName().toString();
-        if (!getMethodName.startsWith("get") || !getMethod.getParameters().isEmpty()) {
+        if (!getMethod.getParameters().isEmpty()) {
           continue;
         }
         TypeMirror typeMirror = getMethod.getReturnType();
         if (typeMirror.getKind() == TypeKind.VOID) {
           continue;
         }
-        String propertyName = Introspector.decapitalize(getMethodName.substring(3));
+
+        String getMethodName = getMethod.getSimpleName().toString();
+        boolean regularGetterName = getMethodName.startsWith("get");
+        boolean booleanGetterName =
+            getMethodName.startsWith("is") && typeMirror.getKind() == TypeKind.BOOLEAN;
```

Comment:
We should also allow `Boolean.class`. I know the JavaBeans spec only mentions `boolean`, but we're not strictly adhering to it.